### ISSUE TITLE
Accessibility bug: nearly impossible to login or sign up using a keyboard from the homepage

### ISSUE THREAD

**Issue by SleeplessByte** [bluesky-social/social-app#6369]
### Steps to Reproduce

1. Go to https://bsky.app/
2. Try to reach the Sign in or Sign up buttons in the bottom bar
3. Good luck, you won't.

I don't think it's the expectation to tab over 400 times to get to these buttons. Additionally, content will load, so you potentially never reach it. The only way to currently sign up or login is to either know the URL or to _google_ (kagi search) the page.

- [ ] Allow users who are not signed in to sign up using a keyboard only without having to tab up to infinity

How to solve? You probably want a skip link, [understanding bypass blocks](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html). Usually these links are [added to skip the navigation](https://webaim.org/techniques/skipnav/), which bluesky lacks as well, but that's a different issue. When logged out, I would add a skip link at the top to "skip to navigation", which is what that is (sign up and login)

### Attachments

| Breakpoint | Focus order |
|--------|-----------------|
| Mobile | ![Focus order overlay on the logged out home page with a sign up reporting as 446, mobile viewport](https://github.com/user-attachments/assets/c439b03d-6f6b-4474-97cd-61e95f9caab5) |
| Desktop | ![Focus order overlay on the logged out home page with a sign up reporting as 416, desktop viewport](https://github.com/user-attachments/assets/e4d04981-9d9c-4ba2-94f2-54f3904e9322)

### What platform(s) does this occur on?

Web (Desktop), Web (Mobile)

### Device Info

Not Relevant

### What version of the app are you using?

Web

### Additional Information

If I recall correctly, Voiceover will not show a number for *each tabstop*, but only for each *visible tabstop*, so this _should not_ be an issue on Android/iOS in terms of using Voice Commands, but it is absolutely an issue for users who use something like a joystick or keyboard to tab through the interactive elements.
